After upgrade xonsh crashes with "Wrong color format" error

## What I did to trigger the issue
I've upgraded with `pip install --upgrade xonsh` to see if another issue was resolved.

## The trace I get:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 21008, in main
    return main_xonsh(args)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 21034, in main_xonsh
    shell.shell.cmdloop()
  File "/usr/local/lib/python3.5/dist-packages/xonsh/ptk/shell.py", line 168, in cmdloop
    line = self.singleline(auto_suggest=auto_suggest)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/ptk/shell.py", line 133, in singleline
    prompt_args['style'] = style_from_dict(DEFAULT_STYLE_DICT)
  File "/usr/local/lib/python3.5/dist-packages/prompt_toolkit/styles/from_dict.py", line 122, in style_from_dict
    attrs = attrs._replace(bgcolor=_colorformat(part[3:]))
  File "/usr/local/lib/python3.5/dist-packages/prompt_toolkit/styles/from_dict.py", line 39, in _colorformat
    raise ValueError('Wrong color format %r' % text)
ValueError: Wrong color format '##ansidarkblue'
Xonsh encountered an issue during launch
Failback to /bin/sh
```
## What I've tried with no success
I've tried un-installing pygments and setting $XONSH_COLOR_STYLE='bw'.

## My current workaround 
I'm starting xonsh with the readline backend (`xonsh --shell-type=readline`).

## xonfig 
```
+------------------+----------------------+
| xonsh            | 0.7.5                |
| Git SHA          | 8ab50b7e             |
| Commit Date      | Aug 16 15:26:52 2018 |
| Python           | 3.5.2                |
| PLY              | 3.9                  |
| have readline    | True                 |
| prompt toolkit   | 1.0.15               |
| shell type       | readline             |
| pygments         | None                 |
| on posix         | True                 |
| on linux         | True                 |
| distro           | Ubuntu               |
| on darwin        | False                |
| on windows       | False                |
| on cygwin        | False                |
| on msys2         | False                |
| is superuser     | False                |
| default encoding | utf-8                |
| xonsh encoding   | utf-8                |
| encoding errors  | surrogateescape      |
+------------------+----------------------+
```

Installing pygments doesn't solve the issue (I had it installed to start with). The trace changes a bit:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 21008, in main
    return main_xonsh(args)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 21034, in main_xonsh
    shell.shell.cmdloop()
  File "/usr/local/lib/python3.5/dist-packages/xonsh/ptk/shell.py", line 168, in cmdloop
    line = self.singleline(auto_suggest=auto_suggest)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/ptk/shell.py", line 131, in singleline
    prompt_args['style'] = PygmentsStyle(pyghooks.xonsh_style_proxy(self.styler))
  File "/usr/local/lib/python3.5/dist-packages/xonsh/pyghooks.py", line 482, in xonsh_style_proxy
    class XonshStyleProxy(Style):
  File "/usr/local/lib/python3.5/dist-packages/pygments/style.py", line 101, in __new__
    ndef[0] = colorformat(styledef)
  File "/usr/local/lib/python3.5/dist-packages/pygments/style.py", line 58, in colorformat
    assert False, "wrong color format %r" % text
AssertionError: wrong color format '##ansidarkblue'
Xonsh encountered an issue during launch
Failback to /bin/sh
```

Here's the output
```
$ xonsh --no-rc
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 21008, in main
    return main_xonsh(args)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 21031, in main_xonsh
    print_welcome_screen()
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 14847, in print_welcome_screen
    print_color(line)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 5103, in print_color
    builtins.__xonsh_shell__.shell.print_color(string, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/ptk/shell.py", line 288, in print_color
    proxy_style = PygmentsStyle(pyghooks.xonsh_style_proxy(self.styler))
  File "/usr/local/lib/python3.5/dist-packages/xonsh/pyghooks.py", line 482, in xonsh_style_proxy
    class XonshStyleProxy(Style):
  File "/usr/local/lib/python3.5/dist-packages/pygments/style.py", line 101, in __new__
    ndef[0] = colorformat(styledef)
  File "/usr/local/lib/python3.5/dist-packages/pygments/style.py", line 58, in colorformat
    assert False, "wrong color format %r" % text
AssertionError: wrong color format '##ansibrown'
Xonsh encountered an issue during launch
```

--- a/news/color_py35.rst
+++ b/news/color_py35.rst
@@ -1,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+* Fixed a potential crash (``AssertionError: wrong color format``) on Python 3.5 and prompt_toolkit 1.
+
+**Security:** None

--- a/xonsh/tools.py
+++ b/xonsh/tools.py
@@ -1734,7 +1734,7 @@ def ansicolors_to_ptk1_names(stylemap):
     modified_stylemap = {}
     for token, style_str in stylemap.items():
         for color, ptk1_color in ANSICOLOR_NAMES_MAP.items():
-            if ptk1_color not in style_str:
+            if '#' + color not in style_str:
                 style_str = style_str.replace(color, ptk1_color)
         modified_stylemap[token] = style_str
     return modified_stylemap

